src/plugin-hooks/useSortBy.js

```diff
@@ -15,6 +15,7 @@ import * as sortTypes from '../sortTypes'
 
 // Actions
 actions.resetSortBy = 'resetSortBy'
+actions.setAllSortBy = 'setAllSortBy'
```

Comment:
How about renaming `setAllSortBy` to `setSortBy`

Reply:
This will be in sync with reset thing...also similar to other plugins (https://github.com/tannerlinsley/react-table/pull/2570/files)

Reply:
Thank you for checking this pull request.
That would be nice. I renamed them.